docs/cluster-deployment/croud.rst

```diff
@@ -0,0 +1,233 @@
+.. _cluster-deployment-croud:
+
+
+===========================
+Deploy a cluster with Croud
+===========================
+
+In this tutorial, we will provide a step-by-step guide to deploying a cluster
+from scratch using our CLI application Croud. Our recommended procedure assumes
+you have completed the :ref:`signup process <sign-up>`. The simplest way to
+then deploy a cluster is to subscribe to CrateDB Cloud using your
+credit card. Here we will explain how to do so, from the initial subscribing to
+the deployment itself. Our payment processing and billing are powered by Stripe.
+
+This tutorial also assumes that you have our CLI tool Croud installed in your
+system. In case not, follow the Croud `Getting started`_ section.
+
+
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+Log in
+======
+
+When working with Croud, the first step is always logging into your account.
+
+Croud, being a CLI application, is operated using commands. Commands should be
+supplemented following this format:
+
+.. code-block:: console
+
+    sh$ croud [COMMAND] [OPTIONS]
+
+To log in execute this command:
+
+.. code-block:: console
+
+    sh$ croud login
+
+This will open a browser window where you will be prompted for your
+credentials. You can also log in using `azuread`, `github`, or `google` using
+the `--idp` argument. See full `Authentication documentation`_ for details.
+
+.. image:: ../_assets/img/croud-login.png
+   :alt: Croud login
+
+After successfully logging in, you will see this prompt in your browser:
+
+.. code-block:: console
+
+    You have successfully logged into CrateDB Cloud!
+    This window can be closed.
+
+Create a project
+================
+
+The next step is to create a project in your organization. It is assumed that
+you created your first organization when signing up.
+
+To create a new project execute this command:
+
+.. code-block:: console
+
+    sh$ croud projects create --name sampleproject
+
+This will create a new project named `sampleproject`. Check out full
+`documentation of projects in Croud`_.
+
+Create a subscription
+=====================
+
+To deploy a cluster, you need to create a subscription first. New subscription
```

Comment:
Unfortunately, AFAIK the only time Subscriptions are actually created is when deploying a cluster - not enough to add a payment method on the billing page :/